seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/constant/EsTypeMappingSeaTunnelType.java

```diff
@@ -32,6 +34,7 @@ public class EsTypeMappingSeaTunnelType {
                     put("string", BasicType.STRING_TYPE);
                     put("keyword", BasicType.STRING_TYPE);
                     put("text", BasicType.STRING_TYPE);
+                    put("binary", BasicType.STRING_TYPE);
```

Comment:
Please use `PrimitiveByteArrayType` to map `binary`

Reply:
+1，`byte[]` is better for `binary` type than `String`

Reply:
ES binary a Base64 encoded string